README.md

```diff
@@ -41,23 +41,25 @@ pip install neural-compressor
 # Install Intel Neural Compressor and TensorFlow
 pip install neural-compressor
 pip install tensorflow
-# Prepare fp32 model
+
+# Prepare the FP32 model by downloading it
 wget https://storage.googleapis.com/intel-optimized-tensorflow/models/v1_6/mobilenet_v1_1.0_224_frozen.pb
-```
-```python
```

Comment:
This example including 2 part, 1st part is bash cmd, use to prepare env, 2nd part is python code need to run with python, so they are not the same level code, should not use the same block.